packages/http-signature-utils/src/test-utils/keys.ts

```diff
@@ -0,0 +1,14 @@
+import crypto from 'crypto'
+import { v4 } from 'uuid'
+
+export type TestKeys = {
+  keyId: string
+  publicKey: crypto.KeyObject
+  privateKey: crypto.KeyObject
+}
+
+export async function generateTestKeys(): Promise<TestKeys> {
+  const keyId = v4()
+  const { privateKey, publicKey } = crypto.generateKeyPairSync('ed25519')
+  return { keyId, privateKey, publicKey }
```

Comment:
```suggestion
  jwk: JWK
  privateKey: crypto.KeyObject
}

export async function generateTestKeys(): Promise<TestKeys> {
  const { privateKey } = crypto.generateKeyPairSync('ed25519')
  return {
    jwk: generateJwk({
      keyId: v4(),
      privateKey
    }),
    privateKey
  }
```

Reply:
Is it worth the effort going through everything again and changing the function calls and subsequent usage of its return values?

Reply:
- https://github.com/interledger/rafiki/issues/845